app/controllers/trainees/programme_details_controller.rb

```diff
@@ -2,8 +2,6 @@
 
 module Trainees
   class ProgrammeDetailsController < ApplicationController
-    before_action :redirect_to_confirm, if: :section_completed?
```

Comment:
Looks like the `def section_completed` in this controller can also be removed now.